global: release checklist

remileduc opened the pull request:
* FIX Adds changes according to release preparation checklist.

Signed-off-by: Rémi Ducceschi <remi.ducceschi@gmail.com>

remileduc commented on INSTALL.rst:
```diff
@@ -15,16 +15,6 @@ this:
-`REST_ENABLE_CORS`    Set to ``True`` to enable Cross-Origin Resource
-                      Sharing. Defaults to ``False``.
-`REST_CORS_RESOURCES` Dictionary for configuring CORS for endpoints.
-                      See Flask-CORS for further details.
```
this variable doesn't exist in the module, I removed it from the doc

drjova replied:
IMHO it might be better to keep them, it'll be confusing for new invenio users. WDYT?

lnielsen replied:
I agree with @drjova that it's better to keep them, so perhaps simply set their value in config.py

remileduc replied:
only the last one has been removed ;)

lnielsen replied:
You're right, actually ``REST_CORS_RESOURCES`` is wrong. It should be ``CORS_RESOURCES`` see https://flask-cors.readthedocs.io/en/latest/api.html#flask_cors.CORS and https://github.com/corydolphin/flask-cors/blob/master/flask_cors/core.py#L33

remileduc replied:
added in the config file here: https://github.com/inveniosoftware/invenio-rest/pull/63/files#diff-944546f76b8cc3d9d71b4cb5316b0ad6R30

remileduc commented on invenio_rest/__init__.py:
```diff
@@ -27,6 +27,74 @@
+    >>> app.register_blueprint(blueprint)
+
+Now you can launch your server and request it on the `/records` endpoint, as
+described in :any:`examplesapp`.
```
it is just a copy of the content of `examples/app.py`

remileduc commented on setup.py:
```diff
@@ -50,6 +50,7 @@
         'Flask-CORS>=2.1.0',
     ],
     'docs': [
+        'dicttoxml>=1.7.4'
```
needed to build the doc because `autodoc` will try to load `examples/app.py`, and fail if `dicttoxml` isn't installed...
This dependency also appear in the `requirement.txt` of the example app.

remileduc commented on setup.py:
```diff
@@ -50,6 +50,7 @@
         'Flask-CORS>=2.1.0',
     ],
     'docs': [
+        'dicttoxml>=1.7.4',
```
This is needed because autodoc will try to load `examples/app.py` and it will fail if `dicttoxml` isn't installed.

This dependency is also described in the `requirement.txt` of the example app.

lnielsen replied:
You can add it as a test dependency since it is the example app which requires it and not only the documentation.

lnielsen commented on setup.py:
```diff
@@ -60,8 +61,8 @@
 
 setup_requires = [
-    'pytest-runner>=2.6.2',
     'Babel>=1.3',
```
The package doesn't seem to have any translation strings so all the translation machinery can be removed including babel packages in setup, transfix config, and babel sections in setup.cfg

remileduc commented:
I've updated the code:
- REST_CORS_RESOURCES has been moved into CORS_RESOURCES
- removed all things related to translations

Travis seems to have some issues when calling `curl` into a test. The result is quite random... maybe we should add a `wait` somewhere to be sure the server is all set up?
